### ISSUE TITLE
Wrong version patch

### ISSUE THREAD

**Issue by oritwoen** [an-anime-team/the-honkers-railway-launcher#60]
For some reason I got wrong version patch downloaded.

ArchLinux, latest bin from AUR.

It should be `1.2.0` patch instead `2.0.1` As a result, the game cannot be launched.

![Zrzut ekranu z 2023-08-03 11-58-50](https://github.com/an-anime-team/the-honkers-railway-launcher/assets/18102267/31ef0fae-69b6-4ef4-9a4d-3a2666afaa9d)

**Comment by krypt0nn** [comment]
No, it is correct 🫥

**Comment by oritwoen** [comment]
You know what @krypt0nn I checked the console. You're right, the problem with starting the game was in something else.

I used to have `gamescope` turned on then I removed it from the system. The button is blocked and gamescope cannot be turned off in the launcher and the console returns its absence. The logic is a bit lame here.

Installing gamescope fixed everything.

![Zrzut ekranu z 2023-08-03 12-11-56](https://github.com/an-anime-team/the-honkers-railway-launcher/assets/18102267/c6748d1e-bed0-42fa-928a-75ea6d358b86)
![Zrzut ekranu z 2023-08-03 12-11-45](https://github.com/an-anime-team/the-honkers-railway-launcher/assets/18102267/69d017e0-f4e2-4b2a-9b4f-253a2896023d)
